app/models/vm_or_template/scanning.rb

```diff
@@ -8,6 +8,12 @@
 module VmOrTemplate::Scanning
   extend ActiveSupport::Concern
 
+  # Smartstate Analysis is unsupported by default.
+  # Subclasses need to override this method if they support SSA.
+  included do
+    supports_not :smartstate_analysis, :reason => _("Smartstate Analysis is not available for VM or Template.")
```

Comment:
@roliveri Marcel suggested he would drop the class description because it's easier to refactor later. 

And also as the user already knows which VM he/she is dealing with, we can skip it. This error message is shown when the button is in disabled state on the UI.

If you disagree, I can add the class description back.